diff --git a/cmake/FindUUID.cmake b/cmake/FindUUID.cmake
index e28ccfbb..bf26039c 100644
--- a/cmake/FindUUID.cmake
+++ b/cmake/FindUUID.cmake
@@ -23,9 +23,9 @@ if (UNIX)
     if(NOT UUID_FOUND)
       include(IgnManualSearch)
       ign_manual_search(UUID
-                        HEADER_NAMES "uuid.h"
-                        LIBRARY_NAMES "uuid libuuid"
-                        PATH_SUFFIXES "uuid")
+                        HEADER_NAMES uuid.h
+                        LIBRARY_NAMES uuid libuuid
+                        PATH_SUFFIXES uuid)
     endif()
 
     # The pkg-config or the manual search will place
