diff --git a/cmake/FindAVDEVICE.cmake b/cmake/FindAVDEVICE.cmake
index c681bf2b..495050f8 100644
--- a/cmake/FindAVDEVICE.cmake
+++ b/cmake/FindAVDEVICE.cmake
@@ -32,6 +32,13 @@ if(NOT AVDEVICE_FOUND)
   if(AVDEVICE_FOUND)
     file(READ "${AVDEVICE_INCLUDE_DIRS}/libavdevice/version.h" ver_file)
 
+    # ffmpeg 5.1 splitted version information in two files
+    # https://github.com/FFmpeg/FFmpeg/commit/884c5976592c2d8084e8c9951c94ddf04019d81d
+    if(EXISTS "${AVDEVICE_INCLUDE_DIRS}/libavdevice/version_major.h")
+      file(READ "${AVDEVICE_INCLUDE_DIRS}/libavdevice/version_major.h" ver_major_file)
+      string(CONCAT ver_file ${ver_file} ${ver_major_file})
+    endif()
+
     string(REGEX MATCH "LIBAVDEVICE_VERSION_MAJOR[ \t\r\n]+([0-9]*)" _ ${ver_file})
     set(ver_major ${CMAKE_MATCH_1})
 
